docs/04-api-reference/front-commerce-remix/defineRemixExtension.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "defineRemixExtension"
+sidebar_position: 4
+description:
+  "defineRemixExtension allow you to extend the features of Front-Commerce for a
+  Remix application."
+---
+
+<p>
+  Extensions are registered in the application using the
+  front-commerce.config.ts file. Read [Register an
+  extension](/docs/remixed/guides/register-an-extension) to learn more.
+</p>
+
+## `defineRemixExtension`
```

Comment:
**nit:** It's a bit redundant with the article's title which is also "`defineRemixExtension`"

Reply:
This is normal to have linkable titles